### ISSUE TITLE
Bug: no-shadow class expression

### ISSUE THREAD

**Issue by helixbass** [helixbass/eslint-plugin-coffee#14]
This is triggering `no-shadow`:
```
a class A
```

**Comment by helixbass** [comment]
Presumably related, this is triggering `no-shadow` and `no-unused-vars`:
```
exports.Rewriter = class Rewriter
```
